scripts/deployments/hatvaultsregistry-deploy.js

```diff
@@ -37,31 +37,38 @@ async function main(config) {
         hatToken = "0x51a6Efc15c50EcE1DaAD1Ee4fbF8DEC76584c365";
     }
 
-    let startBlock = config["rewardController"]["startBlock"];
-    if (!startBlock) {
-        startBlock = await ethers.provider.getBlockNumber();
-    }
-    let epochLength = config["rewardController"]["epochLength"];
-    let epochRewardPerBlock = config["rewardController"]["epochRewardPerBlock"];
-
+    // add rewardcontrollers
     const RewardController = await ethers.getContractFactory("RewardController");
-    const rewardController = await upgrades.deployProxy(RewardController, [
-        hatToken,
-        governance,
-        startBlock,
-        epochLength,
-        epochRewardPerBlock
-    ]);
+    const rewardControllers = [];
+    const rewardControllerImplementations = [];
+    for (const rewardControllerConfig of config.rewardControllers) {
```

Comment:
for (const rewardControllerConfig of config["rewardControllers"]) {
 for consistency :)